pkgs/development/tools/documentation/gtk-doc/default.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
   # Make six available for binaries, python.withPackages creates a wrapper
   # but scripts are not allowed in shebangs so we link it into sys.path.
   postInstall = ''
-    ln -s ${python.pkgs.six}/lib/python2.7/site-packages/* $out/share/gtk-doc/python/
+    ln -s ${python.pkgs.six}/lib/python${python.majorVersion}/site-packages/* $out/share/gtk-doc/python/
```

Comment:
use `${python.sitePackages}`

Reply:
Thanks for the suggestion, I didn't know about that. Updated the PR.